src/devtools/shared/locales/en-US/styleinspector.properties

```diff
@@ -73,6 +73,10 @@ rule.empty=No element selected.
 # rules could not be loaded from the backend.
 rule.notAvailable=Rules not available.
 
+# LOCALIZATION NOTE (rule.noMatching): Text displayed when no selector or rule
+# was matched with rules filtering query.
+rule.noMatching=No matching selector or style.
```

Comment:
feel free to stop using l10n. We're trying to inline all of the strings for simplicity